ql/src/test/results/clientnegative/msck_repair_2.q.out

```diff
@@ -16,7 +16,5 @@ PREHOOK: Output: default@repairtable
 POSTHOOK: query: MSCK TABLE repairtable
 POSTHOOK: type: MSCK
 POSTHOOK: Output: default@repairtable
-PREHOOK: query: MSCK TABLE default.repairtable
-PREHOOK: type: MSCK
-PREHOOK: Output: default@repairtable
-FAILED: Execution Error, return code 1 from org.apache.hadoop.hive.ql.ddl.DDLTask
+#### A masked pattern was here ####
+Query returned non-zero code: -1, cause: null
```

Comment:
this seems to be a different error

Reply:
good catch, i accidentally put a $$ instead of a single $ in msck_repair_2 and msck_repair_3. I'll fix this in the next patch.